libs/index/odc/index/stac.py

```diff
@@ -78,8 +78,11 @@ def _find_self_href(item: Document) -> str:
 
 
 def _get_stac_bands(
-    item: Document, default_grid: str, relative: bool = False,
-    proj_shape: str = None, proj_transform: str = None,
+    item: Document,
+    default_grid: str,
+    relative: bool = False,
+    proj_shape: str = None,
```

Comment:
can't assign `None` into a `str`, should be `Optional[str]`

Reply:
Good point.